### PR TITLE
Improve the build instructions:

### PR DESCRIPTION
**Pull request by dmitry-timofeev**
## Overview
 - Replace transitive link to the build instructions with
 the direct link.
 - Add the library path to the script creating LD_LIBRARY_PATH.
 - Make the instructions more consistent.

---

### Definition of Done

- [x] There are no TODOs left in the code
- [x] The [continuous integration build](https://www.travis-ci.org/exonum/exonum-java-binding) passes

### CODE REVIEW

File: exonum-java-binding-core/rust/ejb-app/TUTORIAL.md [hunk 1]
```diff
@@ -15,19 +17,25 @@ See [Java binding documentation](https://exonum.com/doc/get-started/java-binding
...
 `EJB_ROOT` used in examples of this section corresponds to the Java Binding root directory.
 
-#### LD_LIBRARY_PATH
+#### `LD_LIBRARY_PATH` and `EJB_LIBPATH`
 
 `LD_LIBRARY_PATH` is required to locate native libraries used by Java Binding.
-You need to provide a path to the JVM library (`libjvm.so`) and to the Rust standard library.
+You need to provide paths to:
+  - The JVM library (e.g., `libjvm.so` on Linux).
+  - The Rust standard library that was used to build the application.
```
Comment (Maria-Nosyk) [inline exonum-java-binding-core/rust/ejb-app/TUTORIAL.md]: that is used

File: exonum-java-binding-core/rust/ejb-app/TUTORIAL.md [hunk 2]
```diff
@@ -15,19 +17,25 @@ See [Java binding documentation](https://exonum.com/doc/get-started/java-binding
...
+  - The application libraries used by Java Binding.
```
Comment (Maria-Nosyk) [inline exonum-java-binding-core/rust/ejb-app/TUTORIAL.md]: I would eliminate definite article at the beginning of each list point

File: exonum-java-binding-core/rust/ejb-app/TUTORIAL.md [hunk 3]
```diff
@@ -1,10 +1,12 @@
-This document describes how to configure and run an Exonum node with Java service using Java Binding App.
+# Exonum Java Binding App Tutorial
+This document describes how to configure and run an Exonum node with a Java service using Java Binding App.
 
 ## Prerequisites
 
-See [How to Build section of Contribution Guide][how-to-build].
+Build an application following the instructions in [«How to Build»][how-to-build] section
```
Comment (vitvakatu) [inline exonum-java-binding-core/rust/ejb-app/TUTORIAL.md]: `«»` are not typically used by english speakers.

**Comment by dmitry-timofeev** [comment]
WIPing for it probably needs updates after #336 has landed.

**Pull request merged**